src/Components/Patient/CrudPatient.jsx

```diff
@@ -0,0 +1,12 @@
+import { useContext } from "react";
+import { TablePatient } from "./TablePatient";
+import { GetTheAppContext } from "../../Context/AppContext";
+
+export const CrudPatient = () => {
+  const { getDataAllPatients } = useContext(GetTheAppContext);
+  return (
+    <div>
+      <TablePatient dataTable={getDataAllPatients || []} />
```

Comment:
es posible hacer este cambio?
```suggestion
      <TablePatient dataTable={getAllPatientsData || []} />
```